File: backup_before_industrialisation_20250626/backend/metier_generator/templates/threed/services/threed_asset_service.py
"""
Service métier avancé pour la gestion des assets.
"""

_assets = []


def get_all_assets():
    """Retourne tous les assets."""
    return _assets


def create_asset(data):
    """Crée un nouvel asset et l'ajoute à la liste."""
    asset = dict(id=len(_assets) + 1, **data)
    _assets.append(asset)
    return asset

File: backend/metier_generator/templates/threed/services/threed_asset_service.js
// Service métier avancé pour la gestion des assets
const assets = [];
module.exports = {
  getAllAssets: () => assets,
  createAsset: (data) => {
    const asset = { id: assets.length + 1, ...data };
    assets.push(asset);
    return asset;
  }
};
